README.md

```diff
@@ -1 +1,12 @@
-# Hello-world
\ No newline at end of file
+Hello Microverse Project
```

Comment:
[ OPTIONAL ] Please follow GitHub flow. The main/master branch should be the default branch.